docs/topics/mta-web-reviewing-a-task-log.adoc

```diff
@@ -0,0 +1,17 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-08-01
+:_mod-docs-content-type: PROCEDURE
+
+[id="mta-web-reviewing-a-task-log_{context}"]
+= Reviewing a task log
+
+To find details and logs of a particular {ProductFullName} task, you can use the *Task Manager* page. 
+
+.Procedure
+
+. In the *Migration* view, click *Application inventory*.
```

Comment:
@mpershina the user does not need to click on `Application inventory` menu,  but: "In the **Migration** view, click **Task Manager**"

Reply:
Eh, my bad, copy-paste error :-D